### ISSUE TITLE
Dragging floating panel twice in arrow causes click to go to canvas

### ISSUE THREAD

**Issue by johnsicili** [webstudio-is/webstudio#4682]
1. Open style field with complex value
2. Drag
3. Drag again relatively quickly
4. See click go to canvas

https://github.com/user-attachments/assets/9fe29ccf-1308-49fe-a108-0df33f1c853e

**Comment by kof** [comment]
@johnsicili can't reproduce this

**Comment by johnsicili** [comment]
@kof Can you record your attempt so I can try to see what the difference is?

**Comment by kof** [comment]
https://share.descript.com/view/1gBRgMsJLPZ

**Comment by johnsicili** [comment]
1. Can you also share your navigator? 
2. Maybe repro needs to be followed or defined better. First open the dialog. Second, drag it to the canvas. Third, quickly drag again and see drag status of the component in the navigator:

https://github.com/user-attachments/assets/3a36de0e-e502-4fe0-84c7-30d38994f743

**Comment by kof** [comment]
@johnsicili is it possible that you have some interference with extensions?

**Comment by johnsicili** [comment]
No here is incognito

https://github.com/user-attachments/assets/c2776084-9b89-4794-b2ab-969bc7eff2b0

**Comment by kof** [comment]
I was never able to repro this, are you still able to? maybe something in specific browser versions ....

**Comment by johnsicili** [comment]
Yeah it still happens. Your last video wasn't the exact repro. If you record another I can let you know what's different in our attempts 

> First open the dialog. Second, drag it to the canvas. Third, quickly drag again and see drag status of the component in the navigator:
